src/indexPage/react-components/bot-landing/ModalComponent.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { set as setStorage, remove } from '../../../common/utils/storageManager';
+import { expirationDate, setPopupToken, renderPopup, setTimeOutBanner, setTimeOutPopup } from '../../index';
+import { translate } from '../../../common/i18n';
+
+const setDueDateAgain = () => {
+    remove('setDueDateForBanner');
+    remove('setPopupToken');
+    setStorage('setDueDateForBanner', expirationDate());
+    setStorage('setPopupToken', setPopupToken());
+    renderPopup('close');
+    setTimeOutBanner()
+    setTimeOutPopup()
+}
+
+const ModalComponent = () => {
+    return (
+        <div>
+            <span className='bot-landing-alert-draggable-dialog draggable-dialog'>
```

Comment:
do we need this span wrapped in extra div? can we use div only instead of div and span? 🙂

Reply:
yes we can will update the code.